### PR TITLE
Fix incorrect file references in documentation

### LINKED ISSUE #3961: Reference to incorrect files in migrating-to-systemd.md
### Change Type
[ ] Addition
[X] Correction
[ ] Deprecation
[ ] Cleanup (formatting, typos, etc.)

### Proposed Changes
In `migrating-to-systemd.md`:
- there is a section referring to non existent `contrib/netbox.env`
- references to `gunicorn.conf` should be to `gunicorn.py` 

`3-http-daemon.md` has the correct instructions

### PR DESCRIPTION
**Pull request by mikewright**
### Fixes: #3961 

updated documentation to reference correct files
copied largely from 3-http-daemon.md

### CODE REVIEW

**Review by jeremystretch — CHANGES_REQUESTED** [review]
I'm fine with fixing the mistakes but let's not gut the examples without further discussion. #3961 pertains only to the correction of errors.

File: docs/installation/migrating-to-systemd.md [hunk 1]
```diff
@@ -12,82 +12,22 @@ Migration is not required, as supervisord will still continue to function.
...
 
 ```no-highlight
-# cp contrib/netbox.service /etc/systemd/system/netbox.service
-# cp contrib/netbox-rq.service /etc/systemd/system/netbox-rq.service
+# cp contrib/*.service /etc/systemd/system/
 ```
 
-Edit /etc/systemd/system/netbox.service and /etc/systemd/system/netbox-rq.service. Be sure to verify the location of the gunicorn executable on your server (e.g. `which gunicorn`).  If using CentOS/RHEL.  Change the username from `www-data` to `nginx` or `apache`:
+!!! note
+    These service files assume that gunicorn is installed at `/usr/local/bin/gunicorn`. If the output of `which gunicorn` indicates a different path, you'll need to correct the `ExecStart` path in both files.
```
Comment (kobayashi) [inline docs/installation/migrating-to-systemd.md]: Let us keep the deleted line and add your note lines as supplement.

**Review by kobayashi — CHANGES_REQUESTED** [review]

**Comment by DanSheps** [comment]
Looking at the PR, it looks like @kobayashi suggestion is all that is needed.  The examples that are in there are from the old systemd configuration that we never used so I think we are fine to remove those as part of this.

File: docs/installation/migrating-to-systemd.md [hunk 2]
```diff
@@ -98,3 +38,20 @@ Finally, start the `netbox` and `netbox-rq` services and enable them to initiate
...
+   Active: active (running) since Thu 2019-12-12 19:23:40 UTC; 25s ago
+     Docs: https://netbox.readthedocs.io/en/stable/
+ Main PID: 11993 (gunicorn)
+    Tasks: 6 (limit: 2362)
+   CGroup: /system.slice/netbox.service
+           ├─11993 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+           ├─12015 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+           ├─12016 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+...
+```
```
Comment (DanSheps) [inline docs/installation/migrating-to-systemd.md]: I don't think we need to include this.  Someone who is relatively proficient with systemd will know how to do this and someone who is not won't really know what is going on (beyond the running) and there are other ways to verify that.

File: docs/installation/migrating-to-systemd.md [hunk 3]
```diff
@@ -12,82 +12,22 @@ Migration is not required, as supervisord will still continue to function.
...
-Copy or link contrib/netbox.service and contrib/netbox-rq.service to /etc/systemd/system/netbox.service and /etc/systemd/system/netbox-rq.service
+Copy or link `contrib/netbox.service` and `contrib/netbox-rq.service` to the `/etc/systemd/system/` directory:
 
 ```no-highlight
-# cp contrib/netbox.service /etc/systemd/system/netbox.service
-# cp contrib/netbox-rq.service /etc/systemd/system/netbox-rq.service
+# cp contrib/*.service /etc/systemd/system/
 ```
 
-Edit /etc/systemd/system/netbox.service and /etc/systemd/system/netbox-rq.service. Be sure to verify the location of the gunicorn executable on your server (e.g. `which gunicorn`).  If using CentOS/RHEL.  Change the username from `www-data` to `nginx` or `apache`:
```
Comment (DanSheps) [inline docs/installation/migrating-to-systemd.md]: We may want to keep the username portion, as it is not always the case that www-data is used or available.  However I would just tack it onto the end.

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by DanSheps** [comment]
There has been no activity on this for a few days, I am going to close this out and resolve myself as we have had a few issues opened up in the last few days.

**Pull request closed without merging**